cdk/connectorConfigurationJson.json

```diff
@@ -0,0 +1,14 @@
+{
+  "connector.class": "software.amazon.event.kafkaconnector.EventBridgeSinkConnector",
+  "topics": "notifications",
+  "aws.eventbridge.eventbus.arn": "arn:aws:events:us-east-1:00000000000:event-bus/eventbridge-sink-eventbus",
```

Comment:
assuming account ID will be replaced by the deployment script?

Reply:
Yes, same for region and event bus name